components/camel-couchbase/src/main/java/org/apache/camel/component/couchbase/CouchbaseEndpoint.java

```diff
@@ -225,6 +219,28 @@ public void setPort(int port) {
         this.port = port;
     }
 
+    /**
+     * The collection to use
+     */
+    public String getCollection() {
+        return this.collection;
+    }
+
+    public void setCollection(String collection) {
+        this.collection = collection;
+    }
+
+    public String getScope() {
+        return this.scope;
+    }
+
+    /**
+     * The scope to use
```

Comment:
Do we know from couchbase what scopes is possible? If its like an enum then we can specify that via @UriParam so we have that information

Reply:
I believe you can create custom scopes with custom name .. so it's definitely not a enum.